README.md

```diff
@@ -356,6 +356,17 @@ config.coverage :CI do |coverage|
 end
 ```
 
+### Ignoring Files
+
+Teaspoon allows defining files and directories to ignore when performing coverage instrumentation.
+
+The below example will ensure that the files within `/test/javascripts` and `/vendor/assets` are not intrumented by Istanbul and will not show up in the coverage results.
+
+```ruby
+config.coverage_ignored_files = ["/test/javascripts", "/vendor/assets"]
```

Comment:
Agree here, though those are slightly different. We have an asset, which should respond to logical_path, which gives you the full pathname of the file that's going to be served. I'm unsure if you can do a thing to see if the filepath we have would match that sort of dir glob string.